client/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@safe-global/api-kit": "^3.0.2",
+    "@safe-global/protocol-kit": "^6.0.5",
+    "@safe-global/safe-apps-sdk": "^9.1.0",
+    "@safe-global/types-kit": "^2.0.1",
```

Comment:
I think we don't need these safe-global dependencies. If we use those that mean we did something wrong some where. We want to build our own on-chain tnx service via smart contract, not by using tx service of safe global